src/plugins/data_views/public/services/has_data.ts

```diff
@@ -77,6 +82,41 @@ export class HasData {
     return source;
   };
 
+  private getIndicesViaSearch = async ({
```

Comment:
```suggestion
  private checkIndicesViaSearch = async ({
```


nit because method returns `boolean`, not `Indicies[]`

Reply:
Had it on automerge, will make the name change as part of some other PR. Nice catch :)